ui/app/AppLayouts/Wallet/views/AssetsDetailView.qml

```diff
@@ -20,18 +20,12 @@ import shared.stores 1.0
 Item {
     id: root
 
-    property var token
+    property var token: ""
```

Comment:
If it's indeed an empty JS Object, init it with `({})`, not with an empty string